src/component/NMRium.tsx

```diff
@@ -217,6 +221,10 @@ function InnerNMRium({
     }
   }, [onDataChange, state]);
 
+  useEffect(() => {
+    onViewChange.current?.(state.view);
+  }, [state.view]);
```

Comment:
@stropitek The ref is never updated, so if the callback changes, we won't be calling the new version.